test/unit/test-amp-img.js

```diff
@@ -665,4 +665,45 @@ describe('amp-img', () => {
       );
     });
   });
+  describe('layout intrinsic', () => {
```

Comment:
It would be good to add a test to make sure it expands a floated parent to a CSS defined limit and no larger.

Reply:
Excellent call. Will do that after debugging the Firefox issues.

Reply:
I added two tests to this effect below. Could you PTAL? =)